### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by mattmoor**
Produced via:
  `github.com/client9/misspell`
/assign adrcunha
/cc adrcunha

### CODE REVIEW

File: tools/cleanup/cleanup.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ import (
 
 var (
 	// Authentication method for using Google Cloud Registry APIs.
-	auther = authn.DefaultKeychain
+	author = authn.DefaultKeychain
```
Comment (adrcunha) [inline tools/cleanup/cleanup.go]: This is not author. :)

```suggestion
	auther = authn.DefaultKeychain
```

File: tools/cleanup/cleanup.go [hunk 2]
```diff
@@ -162,7 +162,7 @@ func (d* ImageDeleter) deleteImage(ref string) error {
 	if err != nil {
 		return errors.Wrapf(err, "failed to parse reference %q", ref)
 	}
-	if err := remoteDelete(image, remote.WithAuthFromKeychain(auther)); err != nil {
+	if err := remoteDelete(image, remote.WithAuthFromKeychain(author)); err != nil {
```
Comment (adrcunha) [inline tools/cleanup/cleanup.go]: This is not author. :)

```suggestion
	if err := remoteDelete(image, remote.WithAuthFromKeychain(auther)); err != nil {
```

File: tools/cleanup/cleanup.go [hunk 3]
```diff
@@ -250,15 +250,15 @@ func (d *ImageDeleter) DeleteResources(project string, hoursToKeepResource int,
 			}
 		}
 		return nil
-	}, google.WithAuthFromKeychain(auther))
+	}, google.WithAuthFromKeychain(author))
```
Comment (adrcunha) [inline tools/cleanup/cleanup.go]: This is not author. :)

```suggestion
	}, google.WithAuthFromKeychain(auther))
```

**Comment by mattmoor** [comment]
ugh conflict

**Comment by adrcunha** [comment]
I copied the fixes to #1648, thanks.

**Pull request closed without merging**